react-native/ios/RealmReact/RealmReact.mm

```diff
@@ -69,7 +71,7 @@ - (JSGlobalContextRef)ctx;
         object_setIvar(executor, contextIvar, rctJSContext);
     }
 
-    return [rctJSContext ctx];
+    return [rctJSContext context].JSGlobalContextRef;
```

Comment:
Yes, it has been added quite a while ago: https://github.com/facebook/react-native/commit/45d2c691a3433bd77cc665367c5d37de19aaffcc